ProjectStructureAndRoles.md

```diff
@@ -75,17 +70,17 @@ A Project participant may resign by notifying the team mailing list and by sendi
 
 **Working Group participants**
 
-Each Project participant can be part of one or more of the Working Groups of the Project by adding his name to the WG_PARTICIPANTS.MD files. By that he gets the role Working Group participant.
+Each Project participant can be part of one or more of the Working Groups of the Project by subscribing to the mailing list(s). By that the Project participant gets the role Working Group participant, the WG_PARTICIPANTS.MD file(s) is/are then also adjusted by the administrators of the Project.
 
-A Working Group participant may resign by removing his name of the WG_PARTICIPANTS.MD file.
+A Working Group participant may resign by unsubscribing from the mailing list(s), the WG_PARTICIPANTS.MD file(s) is/are adjusted by the administrators of the Project.
 
 **Contributors**
 
-Each Project participant can contribute to the Sub Projects of the Project. Contributors to a Sub Project can contribute by creating pull requests. There’s no formal process for becoming or resigning as Contributor. The list of Contributors of a specific Sub Project is maintained by GitHub.
+Each Project participant can contribute to the Sub Projects of the Project. Contributors to a Sub Project can contribute by creating pull requests. To become or resign as Contributor please subscribe / unsubscribe to the mailing list of the Sub Project. The list of Contributors of a specific Sub Project is maintained by the administrators of the Project using GitHub functionality.
```

Comment:
Are these 2 related?
1. Contributors to a Sub Project can contribute by creating pull requests. 
2. To become or resign as Contributor please subscribe / unsubscribe to the mailing list of the Sub Project

What stops a user from creating a PR without subscribing to the mailing list?

Reply:
Good point - personally I think anyone creating a PR is a contributor, which is not necessarily the case for everyine subscribed to the mailing list.

Reply:
If someone new creates a PR he/she will be added to the list of Contributors in the GitHub repo automatically. That's how I've understood GitHub. But then we've manually to add him to the mailing and also to the Project participant list. So that should not be the usual process, also as we don't get a notification of this.
It's better if he/she first subscribes to the list (we get a notification) and then create the PR. That should be the defined process.

Reply:
I understand that we would like them to subscribe to mailing list and we can recommend them to do so as a best practice as they will be part of invites/meetings automatically. 
Just that technically this does not make them a contributor, is all.

Reply:
That's correct, if we get the notification from the mailing list we can invite them to be GitHub Contributor. As soon as they've accepted that invitation they can be added within GitHub e.g. as Reviewer to a Pull Request. That should be the standard way doing it.

Reply:
I suppose the confusion is coming from the differendes of the terms "member" (of the organization/project), "collaborator" (someone who has accepted the invitation to a repo and has at least read access in the repo) and "contributor "(everyone who has a merged commit in a repo). See here: https://opensource.stackexchange.com/questions/10106/what-is-the-difference-between-contributor-member-and-collaborator 

If I get the proposed roles correct, we have the following rules - please correct if wrong:

* Every contributor in a repo in the project shall be a participant (= member) of the project, subscribing to the mailing lists etc. I'm not sure if this makes sense ... especially for contributors who just leave a fix somewhere. They can stay "Outside Contributors" in GitHub terms. We should not disallow such contributions (but review them for sure).
* Every collaborator in a repo of the project shall be a participant (= member) of the project*. That make sense from governance perspective. Can even be enforced in Github but not allowing the invitation of "Outside Contributors" as collaborators into a repo.
* Every subscriber of the mailing list will get an invitation as collaborator in the respective repo ... ok, if they don't have to accept the invitation and can still receive the infomation from the mailing list (a subscriber to a LF mailing list has not necessarily a GitHub account)
* Every collaborator within a repo has to be subscribed to the respective mailing list. Personally, I wouldn't request that (there are developers who don't like to be on mailing lists), but acceptable as a policy. I understand that the proposed process relies on this policy.

Reply:
I'll take that point into a new issue to clarify the roles Collaborator vs. Contributor. But I would close it here so that we can close that mailing list thing